support-frontend/assets/helpers/productPrice/paperProductPrices.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import { Monthly } from 'helpers/billingPeriods';
+import { type FulfilmentOptions } from 'helpers/productPrice/fulfilmentOptions';
+import { type PaperProductOptions } from 'helpers/productPrice/productOptions';
+
+import { getCurrency, getCountryGroup, applyPromotion, type ProductPrices, type Price, type Promotion } from './productPrices';
+
+const { name: country } = getCountryGroup('GB');
```

Comment:
jesus christ what in tarnation was i thinking here
```suggestion
const country = getCountryGroup('GB').name;
```